src/main/resources/db/migration/V2__dml.sql

```diff
@@ -274,30 +274,30 @@ values (0, null, 'Bilim', 'Bilim kategorisidir.', 'bilim', 'flask-conical',
         'gelecekbilimde', current_timestamp);
 
 
-insert into gb_user (id, birth_date, email, gender, first_name, last_name, password,
+insert into gb_user (id, birth_date, email, gender, first_name, last_name, username, password,
                      status, role_id, created_by, created_at)
 values ('22afc9b4-807f-4eb2-b286-788631d1ed55', current_date, 'admin@gelecekbilimde.net',
-        'FEMALE', 'Test', 'Yönetici',
+        'FEMALE', 'Test', 'Yönetici', 'yönetici12',
```

Comment:
Daha iyi bir username belirleyebilir miyiz? Türkçe karakter olmaması gerekir diye düşünüyorum. Diğer kullanıcılar için de aynısını yapabilir miyiz?

Reply:
Evet yaparken fark etmemişim teşekkürler